ui/raidboss/data/05-shb/dungeon/akadaemia_anyder.js

```diff
@@ -255,34 +236,16 @@
       'locale': 'fr',
       'replaceSync': {
         'Marquis Morbol': 'Marquis morbol',
-        'laboratory tomato': 'Spécimen de tomate',
-        'laboratory queen': 'Spécimen de reine mandragore',
-        'laboratory onion': 'Spécimen d\'oignon',
-        'laboratory garlic': 'Spécimen d\'ail',
-        'laboratory eggplant': 'Spécimen d\'aubergine',
-        'evil armor': 'Armure maléfique',
-        'engendered soma': 'Soma artificiel',
-        'engendered snapweed': 'Balsamine artificielle',
-        'engendered shark': 'Requin artificiel',
-        'engendered pteraketos': 'Cetus artificiel',
-        'engendered pegasus': 'Pégase artificiel',
-        'engendered monk': 'Poulpe-moine artificiel',
-        'engendered lycopersicum': 'Lycopersicum artificiel',
-        'engendered danbania': 'Dambanha artificiel',
-        'engendered clionid': 'Clionide artificielle',
-        'engendered cactuar giant': 'Gigapampa artificiel',
-        'engendered bombfish': 'Poisson-bombo artificiel',
-        'engendered bomb': 'Bombo artificiel',
-        'engendered bandersnatch': 'Bandersnatch artificiel',
-        'engendered anemone': 'Anémone artificielle',
+        'Evil Armor': 'Armure maléfique',
         'Quetzalcoatl': 'Quetzalcóatl',
         'Doliodus': 'Doliodus',
         'Cladoselache': 'Cladoselache',
-        'Ichthyology': 'département d\'ichtyogénie!',
-        'Phytobiology': 'département de phytogénie!',
-        'Phantomology': 'département de phantasmagénie!',
+        'Ichthyology': 'département d\'ichtyogénie',
+        'Phytobiology': 'département de phytogénie',
+        'Phantomology': 'département de phantasmagénie',
       },
       'replaceText': {
+        '--(?<![^0-9]x)': '--',
```

Comment:
Sorry, can you explain to me what this line is doing?

Reply:
Same problem in Varis_ex request, in the text file, there are these 3 lines:
`183.3 "--targetable--" sync /22:........:Doliodus:........:Doliodus:01/`, this line is in the common.js file. 

`128.7 "--2x targetable--" sync /22:........:Doliodus:........:Doliodus:01/`
`129.7 "--1x targetable--" sync /22:........:Cladoselache:........:Cladoselache:00/`
these 2 lines have an additional digit (1x, 2x). For example in fr, if i delete this line:
`'--(?<![^0-9]x)': '--',`, this creates an error.
![akadaemia_anyder](https://user-images.githubusercontent.com/61565834/84989207-9a6d4900-b143-11ea-96e9-30df9407bb03.PNG)
I saw this code in` hades-ex.js` file, which allows to display the figures when there are in lines.
When checking `python util/check_translation.py -f akadaemia_anyder.js -t fr`, it looks okay.
I have not found any other simpler solution to avoid creating multiple lines of translation.

Reply:
This is the same issue as #1483 in terms of a translation not doing something.  I have left a suggestion over there of what to change.  Let me know what you think.

Reply:
This is changed, thanks for your help.